modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Engine.scala

```diff
@@ -24,11 +24,11 @@ object Engine {
       Engine(q.sequences.mapValues(_.map(f)))
   }
 
-  case class State(conditions: Conditions, sequences: Map[Sequence.Id, Sequence.State])
+  case class State(conditions: Conditions, operator: Option[Operator], sequences: Map[Sequence.Id, Sequence.State])
```

Comment:
Yeah, at some moment it could make sense to group them but I'm not sure when that would be. Probably with three params you want to have a separate class class